sidebar.js

```diff
@@ -1,26 +1,37 @@
+let sidebarTemplateContent
+
+async function loadSidebarTemplate () {
+  const response = await fetch('./sidebar.html')
+  const text = await response.text()
+  const parser = new DOMParser()
+  const doc = parser.parseFromString(text, 'text/html')
+  sidebarTemplateContent = doc.getElementById('sidebar-template').content
```

Comment:
Instead of getting the element by ID, create a `<template>` element and set it's content to the entire `sidebar.html` then append it to the body.